ops/terraform/services/common/s3.tf

```diff
@@ -8,16 +8,6 @@ resource "aws_s3_bucket" "this" {
   }
 }
 
-# block public access to the bucket
-resource "aws_s3_bucket_public_access_block" "this" {
-  bucket = aws_s3_bucket.this.id
-
-  block_public_acls       = true
-  block_public_policy     = true
-  ignore_public_acls      = true
-  restrict_public_buckets = true
-}
-
```

Comment:
I'm not sure if we want to remove these public access blocks? While the `aws_s3_bucket_acl` resources are basically deprecated with the recent default to disabling ACLs altogether, I don't believe this resource is deprecated in the same way. At worst, there's a bit of _explicit_ redundancy with this `aws_s3_bucket_public_access_block`, and I'm not sure what the behavior is if we _remove_ this resource for Buckets that have existed prior to the April change.

Reply:
Thanks! Added back in.